ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -374,6 +374,11 @@ tests:
   steps:
     cluster_profile: packet-sno
     workflow: baremetalds-sno
+- as: e2e-vsphere-assisted
```

Comment:
Maybe create a periodic for this job so we can monitor the failures (if there will be any)

Reply:
will do,  after finishing the development (currently we test nothing just creating a template)